lib/sarif/yarn_audit_sarif.rb

```diff
@@ -0,0 +1,88 @@
+require 'salus/bugsnag'
+module Sarif
+  class YarnAuditSarif < BaseSarif
+    include Salus::SalusBugsnag
+    YARN_URI = 'https://classic.yarnpkg.com/en/docs/cli/audit/'.freeze
+
+    def initialize(scan_report)
+      super(scan_report)
+      @uri = YARN_URI
+      parse_scan_report!
+      @issues = Set.new
+    end
+
+    def parse_scan_report!
+      @logs = @scan_report.to_h.fetch(:logs).dump.split("\\n\\n")
+    rescue KeyError => e
+      bugsnag_notify(e.message)
+      @logs = []
+    end
+
+    def parse_issue(issue)
+      parsed_issue = issue.split('\\n')
```

Comment:
What does `\\n` mean?  Also, I think the separator should be lines that begin with `Package: `.  You might need to do something about white lines/spaces too.

Reply:
I'm guessing this is to parse issues from a line like below?  Or is for something else?
```
"logs": "Package: randomatic\nPatched in: >=3.0.0\nDependency of: babel-jest\nMore info: https://www.npmjs.com/advisories/157\nSeverity: low\nTitl\
e: Cryptographically Weak PRNG\nID: 157\n\nPackage: randomatic\nPatched in: >=3.0.0\nDependency of: jest\nMore info: https://www.npmjs.com/advisories/15\
7\nSeverity: low\nTitle: Cryptographically Weak PRNG\nID: 157\n\nPackage: randomatic\nPatched in: >=3.0.0\nDependency of: stylelint\nMore info: https://\
www.npmjs.com/advisories/157\nSeverity: low\nTitle: Cryptographically Weak PRNG\nID: 157\n\nPackage: hoek\nPatched in: > 4.2.0 < 5.0.0 || >= 5.0.3\nDepe\
ndency of: jest\nMore info: https://www.npmjs.com/advisories/566\nSeverity: moderate\nTitle: Prototype Pollution\nID: 566\n\nPackage: hoek\nPatched in: \
> 4.2.0 < 5.0.0 || >= 5.0.3\nDependency of: webpack\nMore info: https://www.npmjs.com/advisories/566\nSeverity: moderate\nTitle: Prototype Pollution\nID\
: 566\n\nPackage: deep-extend\nPatched in: >=0.5.1\nDependency of: jest\nMore info: https://www.npmjs.com/advisories/612\nSeverity: low\nTitle: Prototyp\
e Pollution\nID: 612\n\nPackage: deep-extend\nPatched in: >=0.5.1\nDependency of: webpack\nMore info: https://www.npmjs.com/advisories/612\nSeverity: lo\
w\n...
```

Reply:
Never mind, ignore the two comments above.
But you probably meant "\\n", not '\n`.  These are not the same.